main.go

```diff
@@ -50,8 +51,9 @@ var (
 
 	persistenceStoragePath = flag.String("storage.local.path", "/tmp/metrics", "Base path for metrics storage.")
 
-	remoteTSDBUrl     = flag.String("storage.remote.url", "", "The URL of the OpenTSDB instance to send samples to.")
-	remoteTSDBTimeout = flag.Duration("storage.remote.timeout", 30*time.Second, "The timeout to use when sending samples to OpenTSDB.")
+	remoteStorageType    = flag.String("storage.remote.type", "opentsdb", "The type of remote storage to use. Valid values: 'opentsdb', 'influxdb'.")
+	remoteStorageURL     = flag.String("storage.remote.url", "", "The URL of the remote storage instance to send samples to.")
```

Comment:
Sounds good. How about then moving the decision whether to do long-term storage from checking whether one of these URLs is set to checking whether `storage.remote.type` is empty, and setting that to an empty string by default?